module3/assignment3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Jquery Themes</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<link rel="stylesheet" href="module3/_css/ctheme1.css"/>
+<link rel="stylesheet" href="module3/_css/ctheme2.css"/>
+<link href="https://fonts.googleapis.com/css?family=Big+Shoulders+Display&display=swap" rel="stylesheet">
+<script src="http://code.jquery.com/jquery-2.1.3.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+
+<!-- Ted Pysh
+	index.html
+	info2439
+	Thoendel
+	9/8/19 
+	-->
+
+</head>
+<body>
+<div data-role="page">
+<div data-role="header">
+<h1>Jquery Mobile Theme roller</h1>
+</div>
+<div data-role="main" class="ui-content">
+<ul data-role="listview">
+<li><a href="module3/_css/ctheme1.css">Monochromatic Theme</a></li>
+<li><a href="">Complementary Theme</a></li>
```

Comment:
These links aren't working. The first one because you are pointing to the .css file rather than the index.html file and the second one doesn't contain a reference.